app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,75 @@
+<h1 class="text-center py-4"><%= @recipe.name.upcase %></h1>
+<div class="d-flex justify-content-between">
+  <div>
+    <p class="lead">Preparation time: <span><%= @recipe.preparation_time %></span></p>
+    <p class="lead">Cooking time :<span><%= @recipe.cooking_time %></span></p>
+    <p class="lead"><%= @recipe.description %></p>
+  </div>
+  <div class="d-flex">
+    <% if @recipe.user == current_user %>
+      <p class="mr-4 m-2"><%= @recipe.public ? 'Pulic' : 'Private' %></p>
+      <%= form_for @recipe, url: recipe_path(@recipe), method: :patch, html: {id: 'recipe-form' } do |f| %>
+        <div class="toggle form-check form-switch m-2">
+          <%= f.check_box :public, { id: 'public-checkbox', class: 'form-check-input', onchange: 'this.form.submit();' } %>
+        </div>
+      <% end %>
+    <%end%>
+  </div>
+</div>
+
+<div class="d-flex justify-content-between mt-5">
+  <button type="button" class="btn btn-success" data-bs-toggle="modal" data-bs-target="#exampleModal">
+    Generate shopping list
+  </button>
+  <button type="button" class="btn btn-primary">
+    <%= link_to 'Add ingredient', new_recipe_recipe_food_path(@recipe), class: "text-white text-decoration-none" %>
+  </button>
+</div>
+
+<hr>
+
+<table class="table table-striped">
+  <thead>
+  <tr>
+    <th>Food</th>
+    <th>Quantity</th>
+    <th>Value per Unit</th>
+    <th>Actions</th>
+  </tr>
+  </thead>
+  <tbody>
+  <% @recipe.recipe_foods.each do |recipe_food| %>
+    <tr>
+      <td><%= recipe_food.food.name %></td>
+      <td><%= recipe_food.quantity %> <%= recipe_food.food.measurement_unit %></td>
+      <td>$ <%= recipe_food.food.price %></td>
+      <td>
+        <%= link_to "Modify", edit_recipe_recipe_food_path(@recipe, recipe_food), class: "btn btn-info text-white" %>
+        <%= link_to "Delete", recipe_recipe_food_path(@recipe, recipe_food), class: "btn btn-danger", method: :delete, data: { "turbo-method": :delete, turbo_confirm: "Are you sure?" } %>
```

Comment:
![image](https://github.com/GhausAliKhan/recipe-app/assets/93039359/e0b44353-601b-42d6-ac66-ef4676faf2cf)

- [ ] It would be nice if you double check the delete button because I am not able to delete recipe-foods from a recipe which I have created :fearful: Kindly fix this issue to make your app work as expected :+1: .